src/shared/components/mutationMapper/MutationMapper.tsx

```diff
@@ -196,6 +196,13 @@ export default class MutationMapper<P extends IMutationMapperProps> extends Reac
                         ) : '-'}
                     </span>
                 </div>
+                <div style={{paddingTop:"1em"}}>
+                    <span data-test="GeneSummaryNrOfExons">{'Number of Exons: '}
```

Comment:
I don't think you need a string variable here, you can simply do something like this:
```suggestion
                    <span data-test="GeneSummaryNrOfExons">
                        Number of Exons: {transcript && transcript.exons && transcript.exons.length > 0? (
                            transcript.exons.length
                        ) : '-'}
                            
```